pkgs/tools/backup/btrbk/default.nix

```diff
@@ -29,24 +41,27 @@ stdenv.mkDerivation rec {
       --replace "/bin/date" "${coreutils}/bin/date" \
       --replace "/bin/echo" "${coreutils}/bin/echo" \
       --replace '$btrbk' 'btrbk'
-
-    # Fix SSH filter script
-    sed -i '/^export PATH/d' ssh_filter_btrbk.sh
-    substituteInPlace ssh_filter_btrbk.sh --replace logger ${utillinux}/bin/logger
   '';
 
   preFixup = ''
     wrapProgram $out/sbin/btrbk \
       --set PERL5LIB $PERL5LIB \
       --prefix PATH ':' "${stdenv.lib.makeBinPath [ btrfs-progs bash mbuffer openssh ]}"
+    substituteInPlace $out/sbin/btrbk --replace "${buildPackages.bash}" "${bash}"
```

Comment:
Why do we have to perform this substitution? i.e., why did the file in `$out` end up with `buildPackages.bash`?

Is it because now bash is in `buildDeps`?

Reply:
The issue is that cross-building will cause `wrapProgram` to run on the build host  and the `@shell@` placeholder with `${buildPackages.bash}` instead of `${pkgs.bash}` which is imo. a bug in `wrapProgram`

Reply:
I see. Yeah this is not very pretty, but I guess we can live with it.